app/src/main/java/br/com/lucas/pomodoroapp/helpers/AlarmManagerHelper.kt

```diff
@@ -0,0 +1,50 @@
+package br.com.lucas.pomodoroapp.helpers
+
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.SystemClock
+import androidx.core.app.AlarmManagerCompat
+import br.com.lucas.pomodoroapp.core.receiver.AlarmReceiver
+import br.com.lucas.pomodoroapp.database.Task
+import dagger.hilt.android.internal.Contexts.getApplication
+import java.util.concurrent.TimeUnit
+
+class AlarmManagerHelper(private val context: Context) {
+    fun setExactAlarm(
+        task: Task
```

Comment:
**#suggestion**
I think `AlarmManagerHelper` doesn't need to knows the `Task`object. Maybe youi could receive just the minutes as an argument.